sambacc/commands/cli.py

```diff
@@ -45,13 +45,11 @@ class Parser(typing.Protocol):
 
     def set_defaults(self, **kwargs: typing.Any) -> None:
         """Set a default value for an argument parser."""
-        ...  # pragma: no cover
```

Comment:
Commit messages says only about ellipsis and not pragma. Should we retain it?

Reply:
The pramga is just there to tell coverage tool not to count the ellipis line when detecting uncovered lines. So that comment is only there because the `...` is there.